stories/dooboo-ui/Checkbox.stories.tsx

```diff
@@ -0,0 +1,179 @@
+import { Checkbox, CheckboxGroup } from '../../main';
+import React, { ReactElement, useState } from 'react';
+
+import { ContainerDeco } from '../../storybook/decorators';
+
+import { storiesOf } from '@storybook/react-native';
+import styled from 'styled-components/native';
+
+const ScrollContainer = styled.ScrollView`
+  width: 100%;
+`;
+
+const Container = styled.View`
+  background-color: transparent;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  flex-direction: column;
+`;
+
+const Title = styled.Text`
+  padding: 20px;
+  font-size: 20px;
+  font-weight: bold;
+`;
+
+const Seperator = styled.View`
+ border-bottom-color: black;
+ border-bottom-width: 1px;
+`;
+
+function DefaultCheckbox(): React.ReactElement {
+  const customStyle = {
+    labelLeft: true,
+  };
+  return (
+    <ScrollContainer>
+
+      <Container>
+        <Title>default</Title>
+        <Checkbox label="defaultChecked" defaultChecked ></Checkbox>
+
+      </Container>
+
+      <Container>
+        <Title>disabled</Title>
+        <Checkbox label="disabled" disabled />
+      </Container>
+
+      <Container>
+        <Title>indeterminate</Title>
+        <Checkbox label="indeterminate" indeterminate />
+      </Container>
+
+      <Container>
+        <Title>labelLeft</Title>
+        <Checkbox label="labelLeft" customStyle={customStyle} />
+      </Container>
+
+    </ScrollContainer>
+  );
+}
+
+function DefaultCheckboxGroup(): React.ReactElement {
+  const plainOptions = ['Apple', 'Pear', 'Orange'];
+  const options = [
+    { label: 'Apple', value: 'Apple' },
+    { label: 'Pear', value: 'Pear' },
+    { label: 'Orange', value: 'Orange' },
+    { label: 'Mango', value: 'Mango' },
+  ];
+  const optionsWithDisabled = [
+    { label: 'Apple', value: 'Apple' },
+    { label: 'Pear', value: 'Pear' },
+    { label: 'Orange', value: 'Orange', disabled: true },
+  ];
+
+  const onChange = (checkedValues): void => {
+    // eslint-disable-next-line no-console
+    console.log('checked = ', checkedValues);
+  };
+  return (
+    <ScrollContainer>
+
+      <Container>
+        <Title>plainOption</Title>
+        <CheckboxGroup options={plainOptions} onChange={onChange} />
+      </Container>
+      <Container>
+        <Title>Default</Title>
+        <CheckboxGroup options={options} onChange={onChange} />
+      </Container>
+
+      <Container>
+        <Title>option with disabled</Title>
+        <CheckboxGroup options={optionsWithDisabled} onChange={onChange} />
+      </Container>
+
+      <Container>
+        <Title>with disabled props</Title>
+        <CheckboxGroup options={options} disabled onChange={onChange} />
+      </Container>
+
+    </ScrollContainer>
+  );
+}
+
+function CheckAll(): React.ReactElement {
```

Comment:
How about `CheckAllExample`? Cause you are using checkAll for state inside.

Reply:
Thank you for the detailed review. We have to work on this part. :)

Reply:
I changed CheckAll to CheckAllExample. :)